app/Jobs/PlatformStatsSummaryJob.php

```diff
@@ -85,8 +85,18 @@ public function prepareStats( array $allStats, $wikis): array {
             }
 
             //add items and properties counts of the wiki to the corresponded arrays
-            array_push($itemsCount, count($this->fetchPagesInNamespace($wiki->domain, self::NAMESPACE_ITEM)));
-            array_push($propertiesCount, count($this->fetchPagesInNamespace($wiki->domain, self::NAMESPACE_PROPERTY)));
+            try {
+                $nextItemCount = count($this->fetchPagesInNamespace($wiki->domain, self::NAMESPACE_ITEM));
+                array_push($itemsCount, $nextItemCount);
+            } catch (\Exception $ex) {
+                Log::warning("Failed to fetch item count for wiki ".$wiki->domain.", will use 0 instead.");
```

Comment:
This is not ideal, but I still think it's better than having this job fail on a single network error or a wiki misbehaving for unknown reasons.

Maybe an idea would be to define a failure budget (maybe 5?) and bail if more than that number of calls failed?